src/chain.rs

```diff
@@ -8,6 +8,7 @@ use revm::primitives::{BlockEnv, SpecId, TxEnv};
 
 use crate::{
     mv_memory::{LazyAddresses, MvMemory},
+    vm::RewardPolicy,
```

Comment:
Let's bring `RewardPolicy` from `vm` to `chain`.

Reply:
ok